buildIncremental.fsx

```diff
@@ -9,6 +9,8 @@ open Fake.Git
 
 module IncrementalTests =   
 
+    let akkaDefaultBranch = "v1.3"
```

Comment:
Probably will keep the default branch `dev` since we won't be working on 2 branches in parallel after `v1.3`.  I'll put an issue in so we don't forget to do switch it to dev as part of the `v1.3 -> dev` merge.